_posts/2021-06-10-solidity-0.8.5-release-announcement.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.5 Release Announcement'
+date: '2021-06-10'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.8.5](https://github.com/ethereum/solidity/releases/tag/v0.8.5) allows conversions
+from ``bytes`` to ``bytesNN`` values, adds the ``verbatim``builtin function to inject
+arbitrary bytecode in Yul and fixes several smaller bugs.
+
+
+## Notable New Features
+### Bytes Conversion
+
+This release introduces the ability to convert ``bytes`` and ``bytes`` slices to fixed bytes types ``bytes1`` / ... / ``bytes32``.
+In case a byte array is longer than the target fixed bytes type, it will be truncated at the end:
+
+```solidity
+function f(bytes calldata c) public view returns (bytes8) {
+	require(c.length == 16, "");
+	bytes8 b = bytes8(c); // c is longer than 8 bytes, so truncation happens
+	return b;
+}
+```
```

Comment:
I'm wondering if it would be better to work with specific examples:
```suggestion
```solidity
function f(bytes calldata c) public view returns (bytes8) {
	return bytes8(c); // If c is longer than 8 bytes, truncation happens
}
```

Calling ``f("1234")`` will return ``"1234"``.

Calling ``f("1234567890")`` will return ``"12345678"``.
```

Reply:
Completely agree, adding